src/System.CommandLine.Subsystems.Tests/AlternateSubsystems.cs

```diff
@@ -1,7 +1,9 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.CommandLine.Directives;
```

Comment:
This is an example of my confusion. If I look at dotnet:main-powderhouse, I see this line present, as well as the files DiagramSubsystemTests.cs, etc.